src/Symfony/Component/Console/Shell.php

```diff
@@ -189,6 +189,8 @@ private function autocompleter($text)
             $command = $this->application->find(substr($text, 0, strpos($text, ' ')));
         } catch (\Exception $e) {
             return true;
+        } catch (\Throwable $e) {
```

Comment:
-1 for that. An Error happening in this code path can only be caused by a bug, and so should bubble to the outside